### PR TITLE
Adview: adapter currency fix.

### PR DESCRIPTION
**Pull request by AdviewOpen**
for the suggestion, fixed the response currency setting.

### CODE REVIEW

**Review by AlexBVolcy — APPROVED** [review]
LGTM

File: adapters/adview/adview.go [hunk 1]
```diff
@@ -100,7 +100,8 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
-	bidResponse.Currency = response.Cur
+	bidResponse.Currency = "USD" //we just support USD for resp
```
Comment (mansinahar) [inline adapters/adview/adview.go:103]: @AdviewOpen Should the `req.Cur` in the request being sent to your exchange needs to be updated as well if your exchange only supports USD? Currently, `req.Cur` would be set to whatever is specified by the publisher. What does your exchange do if the requested currency in `req.Cur` is not USD?

Comment (AdviewOpen) [reply adapters/adview/adview.go:120]: thx and i changed codes and add test script ,pls check it .

File: adapters/adview/adview.go [hunk 2]
```diff
@@ -58,6 +59,22 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		}
 	}
 
+	// Check if imp comes with bid floor amount defined in a foreign currency
+	if imp.BidFloor > 0 && imp.BidFloorCur != "" && strings.ToUpper(imp.BidFloorCur) != "USD" {
+		// Convert to US dollars
+		convertedValue, err := requestInfo.ConvertCurrency(imp.BidFloor, imp.BidFloorCur, "USD")
```
Comment (guscarreon) [inline adapters/adview/adview.go:65]: Pull request #2071 makes possible for adapters to include their own currency conversion rates in their JSON tests. When said PR gets merged, a test case could be included to add test coverage to lies 64 to 72

**Review by guscarreon — APPROVED** [review]
LGTM.

File: adapters/adview/adview.go (hunk 2 above)
Comment (AdviewOpen) [reply adapters/adview/adview.go:65]: i add the case currency_rate_not_found.json, but it seems cause validate error, so i remove it as temporary. so did i forgot something?

**Comment by mansinahar** [comment]
Needs #2071 to be merged in before we can merge this one in

**Review by AlexBVolcy — APPROVED** [review]

**Review by guscarreon — COMMENTED** [review]
Looks good to me @AdviewOpen. Just one more ask: Now that pull request #2071 has been merged, can you please re-incorporate the JSON currency conversion test case you got rid of in a previous commit? JSON tests with currency conversion are now supported. As shown below, file `adapters/adview/adviewtest/supplemental/currency_rate_not_found.json` covers the error scenario in line 67. 

Can we also add a JSON currency test to cover lines 71 and 72?
image

**Comment by AdviewOpen** [comment]
Roger , and test script has been already in git,  great thx for all of you.

**Review by guscarreon — APPROVED** [review]
LGTM. @AdviewOpen we appreciate your response to our feedback

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**